auth0/provider_test.go

```diff
@@ -5,11 +5,49 @@ import (
 	"testing"
 
 	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
 	"github.com/hashicorp/terraform-plugin-sdk/terraform"
 
 	"gopkg.in/auth0.v5/management"
 )
 
+func providerWithWiremock() *schema.Provider {
```

Comment:
While Wiremock is a testing utility we're using, it may not be descriptive at first glance. Would you consider something like `providerWithMockedRequests`, `mockedProvider` or something similar?

Reply:
Updated it to `providerWithTestingConfiguration`.